src/core/ext/filters/channel_idle/legacy_channel_idle_filter.cc

```diff
@@ -176,6 +177,9 @@ void LegacyMaxAgeFilter::PostInit() {
 
   // Start the max age timer
   if (max_connection_age_ != Duration::Infinity()) {
+    auto arena = SimpleArenaAllocator()->MakeArena();
```

Comment:
Similar question here: Isn't it wasteful to create a whole arena for this timer?